Dockerfile

```diff
@@ -1,18 +1,31 @@
-FROM centos:7
-MAINTAINER carcassi@umich.edu
+FROM centos:centos7
 
-RUN rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
-    yum install -y yum-plugin-priorities && \
-    rpm -Uvh http://mirror.grid.uchicago.edu/pub/osg/3.4/osg-3.4-el7-release-latest.rpm
+LABEL maintainer OSG Software <help@opensciencegrid.org>
+
+# Create the squid user with a fixed GID/UID
+RUN groupadd -o -g 10941 squid
+RUN useradd -o -u 10941 -g 10941 -s /sbin/nologin -d /var/lib/squid squid
+
+
+RUN yum -y install http://repo.opensciencegrid.org/osg/3.4/osg-3.4-el7-release-latest.rpm \
+                   epel-release \
+                   yum-plugin-priorities
 
 RUN yum clean all && \
-    yum update -y && \
-    yum install -y frontier-squid && \
-    systemctl enable frontier-squid
+    yum update -y 
+
+RUN yum install -y frontier-squid && \
+    yum install -y supervisor
+
+RUN yum clean all --enablerepo=* && rm -rf /var/cache/yum/
+
 
-COPY app/ /app
+RUN systemctl enable frontier-squid
```

Comment:
What's the point of this? Shouldn't supervisor handle service management?

Reply:
This line was already there in the original Dockerfile... I just moved it down.

No objection in removing it, though.
I don't think it is really necessary.

Please let me know what is your preference.

Reply:
Yeah, let's get rid of this

Reply:
Delete away!